### PR TITLE
CI-342 Remove topper mapping logic

### PR DESCRIPTION
**Pull request by ker-an**
The logic determining which topper to apply to an article has been moved from `o-topper` and into it's own library, [`n-map-content-to-topper`](https://github.com/Financial-Times/n-map-content-to-topper).

We've done this so we can use the `mapContentToTopper` helper on the server-side in `next-article`. Also, it moves us closer to having `o-topper` less tied to Customer Products content.

### CODE REVIEW

**Comment by ker-an** [comment]
Related to https://github.com/Financial-Times/next-article/pull/3934

File: bower.json
Comment (JakeChampion) [inline bower.json:5]: Removing the JavaScript API from this component will mean that it is a breaking change and will need to have a migration guide added to the [migration file](https://github.com/Financial-Times/o-topper/blob/master/MIGRATION.md#migration-guide)

Comment (ker-an) [reply bower.json:5]: Oh okay! I thought it could be a minor release because of what was discussed in this convo? See https://financialtimes.slack.com/archives/C02FU5ARJ/p1597829812214400?thread_ts=1597829441.212200&cid=C02FU5ARJ

Comment (JakeChampion) [reply bower.json:5]: It can be made a minor if we keep the public JavaScript API. We can do that by including the new package as a dependency and having the o-topper `main.js` file re-export the new package. We did that with o-date, which may be useful to look at -- https://github.com/Financial-Times/o-date/blob/master/src/js/date.js#L289-L296

Comment (ker-an) [reply bower.json:5]: OHHHHHH 🤦 Yes, that makes complete sense.

File: package.json
Comment (ker-an) [inline package.json:5]: Update when published to NPM

File: package.json
Comment (JakeChampion) [inline package.json:5]: Unfortunately Origami components need to be developed using Bower dependencies.

`n-map-content-to-topper` is not available on Bower because the GitHub repository is not public. Would you be able to make the repository public?

**Comment by JakeChampion** [comment]
This work is currently blocked on:
1. Making the n-map-content-to-topper repository public
2. Adding a new mapping to the Origami Component Converter to make it work on Bower and npm --> https://github.com/Financial-Times/origami-component-converter/pull/109

File: readme.md
Comment (chee) [inline readme.md:153]: :+1:

File: package.json
Comment (ker-an) [reply package.json:5]: I've made the repo public now 😄 Once the PR in `origami-component-converter` is merged, this will allow me to merge this one?

Comment (JakeChampion) [reply package.json:5]: Some more steps need to be done also:
- We need to add a `bower.json` file to n-map-content-to-topper, I've made a PR for that but the build is failing --> https://github.com/Financial-Times/n-map-content-to-topper/pull/5

- Then we need to run `bower install n-map-content-to-topper --save` in this project and commit that change.

Comment (ker-an) [reply package.json:5]: Great, leave this to me!

Comment (ker-an) [reply package.json]: Hey @JakeChampion, so I read this in the Bower docs so is it correct that I've instead installed the GitHub repo, see https://bower.io/docs/creating-packages. 

Running `bower install n-map-content-to-topper --save` resulted in the following error: `ENOTFOUND Package n-map-content-to-topper not found`.

Comment (JakeChampion) [reply package.json]: Origami runs our own Bower Registry, which contains all public repositories on the Financial-Times and FTLabs GitHub organisations. It looks like it had an issue with n-map-content-to-topper because it was originally made internal. I'll look into this now to fix it :-)

Comment (ker-an) [reply package.json]: Thank you! Let me know when that's done and I'll install using `bower install n-map-content-to-topper --save`

Comment (JakeChampion) [reply package.json]: I've just got it to work and have added it to the bower.json file 👍

File: main.js
Comment (JakeChampion) [inline main.js:1]: This needs to be written using the bower package name which is not namespaced.

When it gets published to npm, Origami uses a [tool](https://github.com/Financial-Times/origami-component-converter) to convert the imports into their npm equivalent. 

Sorry for all these weird quirks, when/if Origami stops supporting Bower, then writing Origami components which use dependencies will be much simpler.

```suggestion
import mapContentToTopper from 'n-map-content-to-topper';
```

Comment (ker-an) [reply main.js:1]: Great, will fix now :)

Comment (ker-an) [reply main.js:1]: And no need to be sorry! I've learned stuff in the process.

**Comment by origamiserviceuser** [comment]
o-topper bundle size difference from 3.0.6 to b11c232bcd4156242545e0e20c8039230fb308cd
js: 1.19kb increase (0.35kb/gzip)
An insignificant difference was also found for: css (master)

**Comment by origamiserviceuser** [comment]
o-topper bundle size difference from 3.0.6 to a5977bcfdcab684de5ff08938c3788b495dabb1a
js: 1.19kb increase (0.35kb/gzip)
An insignificant difference was also found for: css (master)

File: package.json
Comment (chee) [inline package.json:6]:
```suggestion
```

we don't need this here because it's added by [occ](https://github.com/Financial-Times/origami-component-converter/) during release :)

**Review by chee — APPROVED** [review]

File: package.json
Comment (ker-an) [reply package.json:6]: great! I will remove 😄

**Comment by origamiserviceuser** [comment]
o-topper bundle size difference from 3.0.6 to e237f53a9d76e2c4f77c036c38d4d9abec154b04
js: 1.19kb increase (0.35kb/gzip)
An insignificant difference was also found for: css (master)

**Comment by chee** [comment]
nice:)

**Pull request merged**
